src/redux/books/booksSlice.js

```diff
@@ -1,41 +1,103 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
 /* eslint-disable no-param-reassign */
```

Comment:
It is always a good practice not to disable the linters since they are there to help and guide you produce the best standard codes. Kindly remove this and all other linter disabling lines from your code and make sure you do not disable the linters in this and all subsequent projects :pray